public/api/v1/incompatible-mods.json

```diff
@@ -122,5 +122,22 @@
     "tracking": "UNKNOWN",
     "note": "Compostable items cannot be composted on Quilt",
     "icon": "https://cdn.modrinth.com/data/HJetCzWo/baf9ea57b7784e743aa33441105723205d743351.png"
-  }	 	  
+  },
+  {
+    "ids": ["requiem"],
+    "name": "Requiem",
+    "type": "WORKAROUND",
+    "status": "WONT_FIX",
+    "tracking": "https://github.com/Ladysnake/Requiem/issues/670",
+    "note": "Requires a specific version of Quilt Standard Libraries on 1.19.2: https://modrinth.com/mod/qsl/version/4.0.0-beta.30+0.76.0-1.19.2",
+    "icon": "https://cdn.modrinth.com/data/bYmqMxlQ/41b48c13bc4acd691dccc93b09c97995f0509880_96.webp"
+  },
+  {
+    "ids": ["progressive-archery"],
+    "name": "Progressive Archery",
+    "type": "GAME",
+    "status": "UNKNOWN",
+    "tracking": "UNKNOWN",
```

Comment:
would you mind making an issue here?

Reply:
Sure; I'd rather test it manually beforehand, though.